docs/markdown/concepts-and-architecture/polkadot-light-client.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Polkadot Light Client Verifier
+nav_order: 4
+permalink: /concepts/polkadot-light-client-verifier/
+parent: Concepts and Architecture
+has_children: true
+---
+# Polkadot Light Client Verifier
+
+# Overview
+The Polkadot Light Client Verifier for the bridge is a collection of components that allow for trustless verification of bridge messages that come from our bridge parachain on Polkadot and that are sent to Ethereum. This means that it covers trustless verification of:
+ - Polkadot state, ie, changes to the Polkadot Relay Chain which mark new finalized blocks on our bridge parachain
+ - Our parachain state, ie, new parachain blocks
+ - The bridge messages themself, ie, messages that users/applications submit to our bridge parachain for trustless relay
+
+It also needs to do this all while being cheap enough in terms of Ethereum gas prices in order to remain sustainable. There are various components that fit together to make this happen.
+
+## Proof Creation
+We create various custom proofs that will be used for verification on Ethereum.
+
+### Polkadot Relay Chain Proofs
+To facilitate our bridge, the Polkadot relay chain will have a new gadget, BEEFY, added that will produce merkle proofs signed by Polkadot validators that commit to both new Polkadot Relay chain blocks and new parachain blocks in a form that is cheaply verifiable on Ethereum. Producing these commitments will be mandatory for Polkadot validators, and will have additional new slashing conditions associated with them, and so will ensure that verification of and trust in these custom commitments is as strong as direct verification of Polkadot Consensus.
+
+Following these commitments will effectively allow us to follow both (1) new Polkadot Relay chain block headers and (2) new Parachain block headers.
+
+For more details, see [BEEFY](https://github.com/paritytech/grandpa-bridge-gadget){: target="_blank"}
```

Comment:
Rather let the user decide whether to open the link in a new window (Also I think is could possibly be configured globally using the Jekyll config)